### ISSUE TITLE
Trying to access array offset on value of type null

### ISSUE THREAD

**Issue by aleadco** [ticgal/actualtime#82]
<p><a href="https://github.com/ticgal/actualtime/issues/82">https://github.com/ticgal/actualtime/issues/82</a></p><p>Hi how are you getting this error in the logs. Trying to access array offset on value of type null in /var/www/html/marketplace/actualtime/inc/task.class.php on line 414</p><p><br />Created by aleadco</p>
<blockquote>GLPI ticket number: <a href='https://xes.ticgal.com/index.php?redirect=ticket_4335' rel='nofollow'>4335</a><br/>Created with <a href='https://tic.gal'>GitSync</a> in GLPI by Óscar Beiro</blockquote>

**Comment by OscarBeiro** [comment]
Could you please provide more information?


1. Which GLPI and Actualtime versions?
2. Are you upgrading or is it a new install?
3. When is this error arising?

**Comment by jcervantes-sipecom** [comment]
Hi, I'm getting the same error:

> PHP Notice (8): Trying to access array offset on value of type null in /var/www/html/g952/plugins/actualtime/inc/task.class.php at line 414

In differents pages, for example:

- /front/config.form.php?forcetab=PluginActualtimeConfig%241

![image](https://user-images.githubusercontent.com/56606032/97202954-45d97a80-1782-11eb-91d3-1130de3f2241.png)

- Assets > Environment : 

![image](https://user-images.githubusercontent.com/56606032/97201798-dd3dce00-1780-11eb-915e-3eb7124ba722.png)

1. Which GLPI and Actualtime versions?
GLPI 9.5.2 and Actualtime 1.4.0

2. Are you upgrading or is it a new install?
I upgraded from GLPI 9.4.6 and Actualtime 1.3.1 to GLPI 9.5.2 and Actualtime 1.4.0

3. When is this error arising?
When accesing to differents pages like:

- /front/config.form.php?forcetab=PluginActualtimeConfig%241
- Assets > Environment

**Comment by gitsyncuser** [comment]
<p>Hi,</p><p>We are unable to replicate.</p><p>Could you provide your GLPI information?</p><p>Webserver, DB server, PHP version... </p><p>Copy and paste information provide at  Setup > Configuration > System</p><p>Thanks</p><blockquote>Created with <a href='https://tic.gal'>GitSync</a> in GLPI by Óscar Beiro</blockquote>

**Comment by jcervantes-sipecom** [comment]
Of course:

GLPI 9.5.2 (/g952 => /var/www/html/g952) Installation mode: TARBALL
--

Operating system: Linux sg01ilinv3sipe 4.15.0-122-generic #124-Ubuntu SMP Thu Oct 15 13:03:05 UTC 2020 x86_64 PHP 7.4.11 apache2handler (Core, FFI, PDO, PDO_ODBC, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, 	bz2, calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imagick, imap, intl, json, ldap, 	libxml, mbstring, memcache, mysqli, mysqlnd, odbc, openssl, pcre, pdo_mysql, pdo_sqlsrv, posix, pspell, readline, session, 	shmop, snmp, soap, sockets, sodium, sqlsrv, standard, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xml, xmlreader, xmlrpc, 	xmlwriter, xsl, zip, zlib) Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files" 	upload_max_filesize="2M"  Software: Apache/2.4.29 (Ubuntu) (Apache/2.4.29 (Ubuntu) Server at 10.1.3.75 Port 80) 	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36 OPR/71.0.3770.284 (Edition Campaign 34) Server Software: mariadb.org binary distribution 	Server Version: 10.4.15-MariaDB-1:10.4.15+maria~bionic-log 	Server SQL Mode:  	Parameters: glpi@localhost/g9495 	Host info: Localhost via UNIX socket 	 PHP version is at least 7.2.0 - Perfect! Sessions support is available - Perfect! Allocated memory > 64 Mio - Perfect! mysqli extension is installed ctype extension is installed fileinfo extension is installed json extension is installed mbstring extension is installed iconv extension is installed zlib extension is installed curl extension is installed gd extension is installed simplexml extension is installed intl extension is installed ldap extension is installed apcu extension is installed Zend OPcache extension is installed xmlrpc extension is installed CAS extension is installed exif extension is installed zip extension is installed bz2 extension is installed sodium extension is installed Database version seems correct (10.4.15) - Perfect! Timezones seems loaded in database The log file has been created successfully. Write access to /var/www/html/g952/config has been validated. Write access to /var/www/html/g952/files has been validated. Write access to /var/www/html/g952/files/_dumps has been validated. Write access to /var/www/html/g952/files/_sessions has been validated. Write access to /var/www/html/g952/files/_cron has been validated. Write access to /var/www/html/g952/files/_graphs has been validated. Write access to /var/www/html/g952/files/_lock has been validated. Write access to /var/www/html/g952/files/_plugins has been validated. Write access to /var/www/html/g952/files/_tmp has been validated. Write access to /var/www/html/g952/files/_cache has been validated. Write access to /var/www/html/g952/files/_rss has been validated. Write access to /var/www/html/g952/files/_uploads has been validated. Write access to /var/www/html/g952/files/_pictures has been validated. Write access to /var/www/html/g952/marketplace has been validated. Web access to files directory is protected

GLPI_ROOT: /var/www/html/g952 GLPI_CONFIG_DIR: /var/www/html/g952/config GLPI_VAR_DIR: /var/www/html/g952/files GLPI_MARKETPLACE_DIR: /var/www/html/g952/marketplace GLPI_USE_CSRF_CHECK: 1 GLPI_CSRF_EXPIRES: 7200 GLPI_CSRF_MAX_TOKENS: 100 GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org GLPI_INSTALL_MODE: TARBALL GLPI_NETWORK_MAIL: glpi@teclib.com GLPI_NETWORK_SERVICES: https://services.glpi-network.com GLPI_MARKETPLACE_PRERELEASES:  GLPI_USER_AGENT_EXTRA_COMMENTS:  GLPI_AJAX_DASHBOARD: 1 GLPI_CALDAV_IMPORT_STATE: 0 GLPI_DEMO_MODE: 0 GLPI_FORCE_EMPTY_SQL_MODE: 1 GLPI_DOC_DIR: /var/www/html/g952/files GLPI_CACHE_DIR: /var/www/html/g952/files/_cache GLPI_CRON_DIR: /var/www/html/g952/files/_cron GLPI_DUMP_DIR: /var/www/html/g952/files/_dumps GLPI_GRAPH_DIR: /var/www/html/g952/files/_graphs GLPI_LOCAL_I18N_DIR: /var/www/html/g952/files/_locales GLPI_LOCK_DIR: /var/www/html/g952/files/_lock GLPI_LOG_DIR: /var/www/html/g952/files/_log GLPI_PICTURE_DIR: /var/www/html/g952/files/_pictures GLPI_PLUGIN_DOC_DIR: /var/www/html/g952/files/_plugins GLPI_RSS_DIR: /var/www/html/g952/files/_rss GLPI_SESSION_DIR: /var/www/html/g952/files/_sessions GLPI_TMP_DIR: /var/www/html/g952/files/_tmp GLPI_UPLOAD_DIR: /var/www/html/g952/files/_uploads GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/ GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/ GLPI_I18N_DIR: /var/www/html/g952/locales GLPI_VERSION: 9.5.2 GLPI_SCHEMA_VERSION: 9.5.2 GLPI_MIN_PHP: 7.2.0 GLPI_YEAR: 2020 GLPI_MOD_DIR: /var/www/html/g952/plugins/mod

htmlawed/htmlawed version 1.2.5 in (/var/www/html/g952/vendor/htmlawed/htmlawed) phpmailer/phpmailer version 6.1.6 in (/var/www/html/g952/vendor/phpmailer/phpmailer/src) simplepie/simplepie version 1.5.5 in (/var/www/html/g952/vendor/simplepie/simplepie/library) tecnickcom/tcpdf version 6.3.5 in (/var/www/html/g952/vendor/tecnickcom/tcpdf) michelf/php-markdown in (/var/www/html/g952/vendor/michelf/php-markdown/Michelf) true/punycode in (/var/www/html/g952/vendor/true/punycode/src) iamcal/lib_autolink in (/var/www/html/g952/vendor/iamcal/lib_autolink) sabre/dav in (/var/www/html/g952/vendor/sabre/dav/lib/DAV) sabre/http in (/var/www/html/g952/vendor/sabre/http/lib) sabre/uri in (/var/www/html/g952/vendor/sabre/uri/lib) sabre/vobject in (/var/www/html/g952/vendor/sabre/vobject/lib) laminas/laminas-cache in (/var/www/html/g952/vendor/laminas/laminas-cache/src) laminas/laminas-i18n in (/var/www/html/g952/vendor/laminas/laminas-i18n/src) laminas/laminas-serializer in (/var/www/html/g952/vendor/laminas/laminas-serializer/src) monolog/monolog in (/var/www/html/g952/vendor/monolog/monolog/src/Monolog) sebastian/diff in (/var/www/html/g952/vendor/sebastian/diff/src) elvanto/litemoji in (/var/www/html/g952/vendor/elvanto/litemoji/src) symfony/console in (/var/www/html/g952/vendor/symfony/console) scssphp/scssphp in (/var/www/html/g952/vendor/scssphp/scssphp/src) laminas/laminas-mail in (/var/www/html/g952/vendor/laminas/laminas-mail/src/Protocol) laminas/laminas-mime in (/var/www/html/g952/vendor/laminas/laminas-mime/src) rlanvin/php-rrule in (/var/www/html/g952/vendor/rlanvin/php-rrule/src) blueimp/jquery-file-upload in (/var/www/html/g952/vendor/blueimp/jquery-file-upload/server/php) ramsey/uuid in (/var/www/html/g952/vendor/ramsey/uuid/src) psr/log in (/var/www/html/g952/vendor/psr/log/Psr/Log) psr/simple-cache in (/var/www/html/g952/vendor/psr/simple-cache/src) mexitek/phpcolors in (/var/www/html/g952/vendor/mexitek/phpcolors/src/Mexitek/PHPColors) guzzlehttp/guzzle in (/var/www/html/g952/plugins/oauthimap/vendor/guzzlehttp/guzzle/src) guzzlehttp/psr7 in (/var/www/html/g952/plugins/oauthimap/vendor/guzzlehttp/psr7/src) wapmorgan/unified-archive in (/var/www/html/g952/vendor/wapmorgan/unified-archive/src) paragonie/sodium_compat in (/var/www/html/g952/vendor/paragonie/sodium_compat/src) phpCas version 1.3.3 in (/usr/share/php)

accounts             Name: Accounts                       Version: 2.6.0      State: Enabled 	activity             Name: Activities                     Version: 3.0.0      State: Enabled 	actualtime           Name: ActualTime                     Version: 1.4.0      State: Enabled 	fields               Name: Additional Fields              Version: 1.11.0     State: Enabled 	advancedplanning     Name: advancedplanning               Version: 0.2.0      State: Enabled 	news                 Name: Alerts                         Version: 1.7.0      State: Enabled 	archisw              Name: Apps structures                Version: 2.2.1      State: Enabled 	cleanarchivedemails  Name: Archived eMail clean           Version: 1.3.1      State: Enabled 	impacts              Name: Asset impacts                  Version: 1.3.7      State: Installed / not activated 	badges               Name: Badges                         Version: 2.6.0      State: Enabled 	barcode              Name: Barcode                        Version: 2.5.0      State: Enabled 	behaviors            Name: Behaviours                     Version: 2.4.1      State: Enabled 	positions            Name: Cartography                    Version: 4.6.0      State: Installed / not activated 	childticketmanager   Name: Child ticket manager           Version: 2.1.3      State: Enabled 	cmdb                 Name: CMDB                           Version: 2.2.1      State: Enabled 	consumables          Name: Consumable request             Version: 1.5.0      State: Installed / not activated 	costs                Name: Costs                          Version: 1.3.0      State: Enabled 	credit               Name: Credit vouchers                Version: 1.8.0      State: Enabled 	dashboard            Name: Dashboard                      Version: 0.9.9      State: Not installed 	datainjection        Name: Data injection                 Version: 2.8.0      State: Enabled 	databases            Name: Databases                      Version: 2.3.0      State: Enabled 	dataflows            Name: Dataflows                      Version: 2.2.1      State: Enabled 	archimap             Name: Diagrams                       Version: 2.2.1      State: Enabled 	domains              Name: Domains                        Version: 2.2.1      State: Enabled 	manageentities       Name: Entities portal                Version: 3.2.0      State: Enabled 	environment          Name: Environment                    Version: 2.4.1      State: Enabled 	escalade             Name: Escalation                     Version: 2.6.0      State: Enabled 	father               Name: Father&Sons                    Version: 1.2.0RC2   State: Enabled 	financialreports     Name: Financial reports              Version: 2.6.0      State: Installed / not activated 	formcreator          Name: Form Creator                   Version: 2.10.2     State: Enabled 	formvalidation       Name: Form Validation                Version: 0.6.8      State: Installed / not activated 	archifun             Name: Functional Areas               Version: 2.2.1      State: Enabled 	fusioninventory      Name: FusionInventory                Version: 9.5.0+1.0  State: Enabled 	gdrive               Name: GDrive                         Version: 1.3.0      State: Enabled 	airwatch             Name: GLPi Airwatch Connector        Version: 1.5.0      State: Enabled 	dev                  Name: GLPI Development Helper        Version: 1.0.0      State: Enabled 	mod                  Name: GLPI Modifications             Version: 2.0.2      State: Enabled 	resources            Name: Human Resources                Version: 2.7.0      State: Enabled 	sccm                 Name: Interface - SCCM               Version: 2.2.1      State: Not installed 	geninventorynumber   Name: Inventory number generation    Version: 2.5.0      State: Enabled 	addressing           Name: IP Adressing                   Version: 2.9.0      State: Enabled 	uninstall            Name: Item's Lifecycle (uninstall)   Version: 2.7.0      State: Enabled 	itilcategorygroups   Name: ItilCategory Groups            Version: 2.4.0      State: Enabled 	jamf                 Name: JAMF Plugin for GLPI           Version: 2.1.2      State: Enabled 	jsaddons             Name: JS Addons                      Version: 1.0.0      State: Enabled 	livechat             Name: Livechat                       Version: 1.0.4      State: Enabled 	mailanalyzer         Name: Mail Analyzer                  Version: 1.6.1      State: Enabled 	mantis               Name: MantisBT synchronization       Version: 4.4.0      State: Not installed 	mask                 Name: Mask                           Version: 2.3.0      State: Enabled 	metademands          Name: Meta-Demands                   Version: 2.7.2      State: Enabled 	metabase             Name: metabase                       Version: 1.2.0      State: Enabled 	mreporting           Name: More Reporting                 Version: 1.7.0      State: Enabled 	satisfaction         Name: More satisfaction              Version: 1.5.0      State: Enabled 	moreticket           Name: More ticket                    Version: 1.6.0      State: Enabled 	mydashboard          Name: My Dashboard                   Version: 1.8.0      State: Not installed 	notifications        Name: Notifications                  Version: 9.4+1.0    State: Enabled 	oauthimap            Name: Oauth IMAP                     Version: 1.0.0      State: Enabled 	genericobject        Name: Objects management             Version: 2.9.0      State: Enabled 	ocsinventoryng       Name: OCS Inventory NG               Version: 1.7.0      State: Enabled 	officeonline         Name: Office Online                  Version: 1.3.2      State: Installed / not activated 	order                Name: Orders management              Version: 2.6.0      State: Enabled 	additionalalerts     Name: Others alerts                  Version: 2.2.0      State: Installed / not activated 	phpsaml              Name: PHP SAML                       Version: 1.0.0      State: Enabled 	pdf                  Name: Print to pdf                   Version: 1.7.0      State: Enabled 	printercounters      Name: Printer counters               Version: 1.7.0      State: Enabled 	processmaker         Name: Process Maker                  Version: 3.6.5      State: Installed / not activated 	projectbridge        Name: Projectbridge                  Version: 2.3RC2     State: Enabled 	protocolsmanager     Name: Protocols manager              Version: 1.4        State: Enabled 	releases             Name: Releases                       Version: 1.0.0      State: Enabled 	reports              Name: Reports                        Version: 1.14.0     State: Enabled 	reservation          Name: Reservation                    Version: 2.2.0      State: Enabled 	screenshot           Name: Screenshot                     Version: 1.0.0      State: Enabled 	shellcommands        Name: Shell Commands                 Version: 2.3.0      State: Installed / not activated 	statecheck           Name: Statecheck Rules               Version: 2.2.1      State: Not installed 	manufacturersimports Name: Suppliers imports              Version: 2.3.0      State: Enabled 	tag                  Name: Tag Management                 Version: 2.8.0      State: Enabled 	taskdrop             Name: TaskDrop                       Version: 1.2.0      State: Enabled 	tasklists            Name: Tasks list                     Version: 1.6.0      State: Enabled 	ticketcleaner        Name: Ticket Cleaner                 Version: 2.5.1      State: Enabled 	timelineticket       Name: Timeline of tickets            Version: 9.4+1.0    State: Installed / not activated 	timezones            Name: Timezones                      Version: 2.5.2      State: Installed / not activated 	treeview             Name: Tree view                      Version: 1.9.0      State: Enabled 	typology             Name: Typologies                     Version: 3.0.0      State: Enabled 	useditemsexport      Name: Used items export              Version: 2.4.0      State: Enabled 	webapplications      Name: Web applications               Version: 3.0.0      State: Enabled 	webresources         Name: Web Resources                  Version: 1.3.0      State: Enabled 	xivo                 Name: xivo                           Version: 1.0.0      State: Enabled 	yagp                 Name: yagp                           Version: 1.1.0      State: Enabled

**Comment by xacobofg** [comment]
Hi.
Leave only Actualtime enabled and check if it still gives an error, to see if the error is due to another plugin

**Comment by jcervantes-sipecom** [comment]
OK, I did what you said, and the error persist:

![image](https://user-images.githubusercontent.com/56606032/97347763-3aa64d80-185b-11eb-8dd9-0016370a91ff.png)

As you can see, only actualtime plugin is enabled:

> accounts             Name: Accounts                       Version: 2.6.0      State: Installed / not activated
	activity             Name: Activities                     Version: 3.0.0      State: Installed / not activated
	actualtime           Name: ActualTime                     Version: 1.4.0      State: Enabled
	fields               Name: Additional Fields              Version: 1.11.0     State: Installed / not activated
	advancedplanning     Name: advancedplanning               Version: 0.2.0      State: Installed / not activated
	news                 Name: Alerts                         Version: 1.7.0      State: Installed / not activated
	archisw              Name: Apps structures                Version: 2.2.1      State: Installed / not activated
	cleanarchivedemails  Name: Archived eMail clean           Version: 1.3.1      State: Installed / not activated
	impacts              Name: Asset impacts                  Version: 1.3.7      State: Not installed
	badges               Name: Badges                         Version: 2.6.0      State: Installed / not activated
	barcode              Name: Barcode                        Version: 2.5.0      State: Installed / not activated
	behaviors            Name: Behaviours                     Version: 2.4.1      State: Installed / not activated
	positions            Name: Cartography                    Version: 4.6.0      State: Not installed
	childticketmanager   Name: Child ticket manager           Version: 2.1.3      State: Installed / not activated
	cmdb                 Name: CMDB                           Version: 2.2.1      State: Installed / not activated
	consumables          Name: Consumable request             Version: 1.5.0      State: Not installed
	costs                Name: Costs                          Version: 1.3.0      State: Installed / not activated
	credit               Name: Credit vouchers                Version: 1.8.0      State: Installed / not activated
	dashboard            Name: Dashboard                      Version: 0.9.9      State: Not installed
	datainjection        Name: Data injection                 Version: 2.8.0      State: Installed / not activated
	databases            Name: Databases                      Version: 2.3.0      State: Installed / not activated
	dataflows            Name: Dataflows                      Version: 2.2.1      State: Installed / not activated
	archimap             Name: Diagrams                       Version: 2.2.1      State: Installed / not activated
	domains              Name: Domains                        Version: 2.2.1      State: Installed / not activated
	manageentities       Name: Entities portal                Version: 3.2.0      State: Installed / not activated
	environment          Name: Environment                    Version: 2.4.1      State: Installed / not activated
	escalade             Name: Escalation                     Version: 2.6.0      State: Installed / not activated
	father               Name: Father&Sons                    Version: 1.2.0RC2   State: Installed / not activated
	financialreports     Name: Financial reports              Version: 2.6.0      State: Not installed
	formcreator          Name: Form Creator                   Version: 2.10.2     State: Installed / not activated
	formvalidation       Name: Form Validation                Version: 0.6.8      State: Not installed
	archifun             Name: Functional Areas               Version: 2.2.1      State: Not installed
	fusioninventory      Name: FusionInventory                Version: 9.5.0+1.0  State: Installed / not activated
	gdrive               Name: GDrive                         Version: 1.3.0      State: Installed / not activated
	airwatch             Name: GLPi Airwatch Connector        Version: 1.5.0      State: Installed / not activated
	dev                  Name: GLPI Development Helper        Version: 1.0.0      State: Installed / not activated
	mod                  Name: GLPI Modifications             Version: 2.0.2      State: Installed / not activated
	resources            Name: Human Resources                Version: 2.7.0      State: Installed / not activated
	sccm                 Name: Interface - SCCM               Version: 2.2.1      State: Not installed
	geninventorynumber   Name: Inventory number generation    Version: 2.5.0      State: Installed / not activated
	addressing           Name: IP Adressing                   Version: 2.9.0      State: Installed / not activated
	uninstall            Name: Item's Lifecycle (uninstall)   Version: 2.7.0      State: Installed / not activated
	itilcategorygroups   Name: ItilCategory Groups            Version: 2.4.0      State: Installed / not activated
	jamf                 Name: JAMF Plugin for GLPI           Version: 2.1.2      State: Installed / not activated
	jsaddons             Name: JS Addons                      Version: 1.0.0      State: Installed / not activated
	livechat             Name: Livechat                       Version: 1.0.4      State: Installed / not activated
	mailanalyzer         Name: Mail Analyzer                  Version: 1.6.1      State: Installed / not activated
	mantis               Name: MantisBT synchronization       Version: 4.4.0      State: Not installed
	mask                 Name: Mask                           Version: 2.3.0      State: Installed / not activated
	metademands          Name: Meta-Demands                   Version: 2.7.2      State: Installed / not activated
	metabase             Name: metabase                       Version: 1.2.0      State: Installed / not activated
	mreporting           Name: More Reporting                 Version: 1.7.0      State: Installed / not activated
	satisfaction         Name: More satisfaction              Version: 1.5.0      State: Installed / not activated
	moreticket           Name: More ticket                    Version: 1.6.0      State: Installed / not activated
	mydashboard          Name: My Dashboard                   Version: 1.8.0      State: Not installed
	notifications        Name: Notifications                  Version: 9.4+1.0    State: Installed / not activated
	oauthimap            Name: Oauth IMAP                     Version: 1.0.0      State: Installed / not activated
	genericobject        Name: Objects management             Version: 2.9.0      State: Installed / not activated
	ocsinventoryng       Name: OCS Inventory NG               Version: 1.7.0      State: Installed / not activated
	officeonline         Name: Office Online                  Version: 1.3.2      State: Not installed
	order                Name: Orders management              Version: 2.6.0      State: Installed / not activated
	additionalalerts     Name: Others alerts                  Version: 2.2.0      State: Not installed
	phpsaml              Name: PHP SAML                       Version: 1.0.0      State: Installed / not activated
	pdf                  Name: Print to pdf                   Version: 1.7.0      State: Installed / not activated
	printercounters      Name: Printer counters               Version: 1.7.0      State: Installed / not activated
	processmaker         Name: Process Maker                  Version: 3.6.5      State: Not installed
	projectbridge        Name: Projectbridge                  Version: 2.3RC2     State: Installed / not activated
	protocolsmanager     Name: Protocols manager              Version: 1.4        State: Installed / not activated
	releases             Name: Releases                       Version: 1.0.0      State: Installed / not activated
	reports              Name: Reports                        Version: 1.14.0     State: Installed / not activated
	reservation          Name: Reservation                    Version: 2.2.0      State: Installed / not activated
	screenshot           Name: Screenshot                     Version: 1.0.0      State: Installed / not activated
	shellcommands        Name: Shell Commands                 Version: 2.3.0      State: Not installed
	statecheck           Name: Statecheck Rules               Version: 2.2.1      State: Not installed
	manufacturersimports Name: Suppliers imports              Version: 2.3.0      State: Installed / not activated
	tag                  Name: Tag Management                 Version: 2.8.0      State: Installed / not activated
	taskdrop             Name: TaskDrop                       Version: 1.2.0      State: Installed / not activated
	tasklists            Name: Tasks list                     Version: 1.6.0      State: Installed / not activated
	ticketcleaner        Name: Ticket Cleaner                 Version: 2.5.1      State: Installed / not activated
	timelineticket       Name: Timeline of tickets            Version: 9.4+1.0    State: Not installed
	timezones            Name: Timezones                      Version: 2.5.2      State: Installed / not activated
	treeview             Name: Tree view                      Version: 1.9.0      State: Installed / not activated
	typology             Name: Typologies                     Version: 3.0.0      State: Installed / not activated
	useditemsexport      Name: Used items export              Version: 2.4.0      State: Installed / not activated
	webapplications      Name: Web applications               Version: 3.0.0      State: Installed / not activated
	webresources         Name: Web Resources                  Version: 1.3.0      State: Installed / not activated
	xivo                 Name: xivo                           Version: 1.0.0      State: Installed / not activated
	yagp                 Name: yagp                           Version: 1.1.0      State: Installed / not activated

This is the backtrace of the error:

> glpiphplog.NOTICE:   *** PHP Notice (8): Trying to access array offset on value of type null in /var/www/html/g952/plugins/actualtime/inc/task.class.php at line 414
  Backtrace :
  plugins/actualtime/ajax/timer.php:225              PluginActualtimeTask::getTask()

Also, this error was in a previous version, just different line:

- GLPI 9.4.6 plugin version 1.3.1
- URL: /front/entity.form.php?id=0
-- In all tab: Entities, Address, Notes, etc.

![image](https://user-images.githubusercontent.com/56606032/97347966-82c57000-185b-11eb-80c2-8ef0d72d2703.png)

**Comment by xacobofg** [comment]
Apply this patch and check if it still gives an error
https://github.com/ticgal/actualtime/commit/c73216e8a6c337e62c3b6e77e3fb31e30f230ab8.patch

**Comment by jcervantes-sipecom** [comment]
I confirm this patch was the solution for that error for plugin version 1.4.0 :

![image](https://user-images.githubusercontent.com/56606032/97451755-0df94100-1902-11eb-9fa4-950c3550a77e.png)

For plugin 1.3.1 I had to manually change the line 403 and 404 for:

```
//$row=$req->next();
//      return $row['tasks_id'];
if($row=$req->next()){
       return $row['tasks_id'];
     }else{
        return 0;
     }
```

**Comment by aleadco** [comment]
Sorry I haven't had much time. I see that they have taken action, I have applied the patch and for me it has a solution

**Comment by aleadco** [comment]
For my part, this ticket can be closed if the developer considers it.  A greeting.

**Comment by OscarBeiro** [comment]
Fixed on new release 1.5.0
